app/scripts/controllers/map.js

```diff
@@ -17,62 +17,102 @@ app.controller('MapCtrl', [
   'Map',
   'BaseMap',
   function ($scope, $http, $routeParams, $timeout, Map, BaseMap) {
-    var geoserverUrl = 'http://localhost:8080/geoserver/wms';
-    $scope.layers = {};
+      var geoserverUrl = 'http://localhost:8080/geoserver/wms';
```

Comment:
Why is this whitespace shift here?

(Nitpick but it's nice to eliminate whitespace commits if we possibly can, is your `.editorconfig` solid?)